functions.opengraph.php

```diff
@@ -152,16 +152,17 @@ function jetpack_og_tags() {
 			$tags['og:description'] = wp_kses( trim( convert_chars( wptexturize( $tags['og:description'] ) ) ), array() );
 		}
 
-		$tags['article:published_time'] = date( 'c', strtotime( $data->post_date_gmt ) );
-		$tags['article:modified_time']  = date( 'c', strtotime( $data->post_modified_gmt ) );
+		$tags['article:published_time'] = gmdate( 'c', strtotime( $data->post_date_gmt ) );
```

Comment:
Did you make that change to address another issue? It may be best to move it to its own Pull Request, since it's not really related to this issue.

Reply:
Yes, I was getting a PHPCS error:

 ```
155 | ERROR   | date() is affected by runtime timezone changes which
     |         | can cause date/time to be incorrectly displayed. Use
     |         | gmdate() instead.
     |         | (WordPress.DateTime.RestrictedFunctions.date_date)
 156 | ERROR   | date() is affected by runtime timezone changes which
     |         | can cause date/time to be incorrectly displayed. Use
     |         | gmdate() instead.
```

Should I ignore those and use the `--no-verify` flag when committing?

Reply:
Okay. In this case let's leave this in. Thanks!